dashboard/v1.1/src/pages/Input/Input.tsx

```diff
@@ -32,26 +44,55 @@ interface AlertSnackBar {
   message: string;
 }
 
-const requestsTypeSupported = ['get', 'post', 'put', 'delete', 'patch'];
-
-const hyperTexts = ['https', 'http', 'manual'];
-
 function Alert(props: AlertProps) {
   return <MuiAlert elevation={6} variant="filled" {...props} />;
 }
 
+const useStyles = makeStyles(theme => ({
```

Comment:
How about a rather modular approach for each page?
Each page has the subdirectories:
- `PageName` - the entrypoint to the page.
- `styles` - contains all the css and styles.
- `utils` - parsers, handlers, types, etc.
- `components` - seggregated individual components imported into the main page.

Reply:
This looks great way to do. However, I do worry of styles being static, so we might need to implement webpack to combine contents dynamically in runtime. But given that it will complicate the process based on the current scale, we can keep styles in a single file but go with the structure you mentioned above (excluding styles).